pkg/kf/commands/apps/push_test.go

```diff
@@ -182,6 +182,46 @@ func TestPushCommand(t *testing.T) {
 			},
 			wantErr: errors.New("malformed environment variable: invalid"),
 		},
+		"container image": {
+			namespace: "some-namespace",
+			args: []string{
+				"app-name",
+				"--docker-image", "some-image",
```

Comment:
Can we also add some environment variables to ensure the aforementioned problem is addressed?

Reply:
preferably as a separate test so we don't test two things at the same time :-)

Reply:
done